cli/cli.cc

```diff
@@ -93,6 +93,9 @@ static auto constexpr Options = std::array<tr_option, 19>{
       { 'v', "verify", "Verify the specified torrent", "v", false, nullptr },
       { 'V', "version", "Show version number and exit", "V", false, nullptr },
       { 'w', "download-dir", "Where to save downloaded data", "w", true, "<path>" },
+      { 500, "sequential-download", "Download pieces sequentialy", "seq", 0, nullptr },
+      { 501, "random-download", "Download pieces randomly (default)", "rnd", 0, nullptr },
```

Comment:
FTBFS

```
/usr/home/user/buildAgent2/work/1f3bff43dd55cfe9/cli/cli.cc:97:7: error: excess elements in array initializer
[09:37:31 ](https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_FreeBSDAmd64polarssl/339077?expandchucktcpluginBuild=true&expandPull+Request+Details=true&expandBuildProblemsSection=true&hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&showLog=339077_1134_1134&logView=flowAware)        { 501, "random-download", "Download pieces randomly (default)", "rnd", 0, nullptr },
[09:37:31 ](https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_FreeBSDAmd64polarssl/339077?expandchucktcpluginBuild=true&expandPull+Request+Details=true&expandBuildProblemsSection=true&hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&showLog=339077_1135_1135&logView=flowAware)        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[09:37:31 ](https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_FreeBSDAmd64polarssl/339077?expandchucktcpluginBuild=true&expandPull+Request+Details=true&expandBuildProblemsSection=true&hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&showLog=339077_1136_1136&logView=flowAware)  1 error generated.
```

I think the std::array size on line 71 needs to be increased by 2 for these :smile_cat:

Reply:
done - my local build was completely fine without this change - must be a cache oddity :thinking: